modules/twinklearv.js

```diff
@@ -28,6 +28,8 @@ Twinkle.arv = function twinklearv() {
 	}, 'ARV', 'tw-arv', 'Report ' + userType + ' to administrators');
 };
 
+Twinkle.arv.temporaryAccountNotice = 'You are reporting a new temporary account. These accounts, with usernames like ~2025-12345-67, have replaced IP addresses. Before reporting any temporary accounts, please familiarize yourself with [[Wikipedia:Temporary accounts|how they work]].';
```

Comment:
I'm not sure we need a global variable for this. Any objection to inlining this on line 96?